game/plugin/skills/runecrafting/src/data.kt

```diff
@@ -0,0 +1,115 @@
+import javafx.geometry.Pos
```

Comment:
JavaFX isn't bundled with distributions of the OpenJDK.  We shouldn't use any classes from it.

Reply:
I did not realize that this was imported, I do not believe that I am using it. Thanks.